indico/modules/events/editing/operations.py

```diff
@@ -136,6 +140,7 @@ def confirm_editable_changes(revision, submitter, action, comment):
     if action == EditingConfirmationAction.accept:
         publish_editable_revision(revision)
     db.session.flush()
+    notify_submitter_confirmation(revision)
```

Comment:
If the submitter rejects the changes the email you are sending here is rather misleading. I'd send a different one mentioning that the submitter rejected the editor's changed in case of `EditingConfirmationAction.reject`.